buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy

```diff
@@ -38,6 +40,10 @@ abstract class AndroidBuckRuleComposer extends JvmBuckRuleComposer {
         return "bin_${target.name}"
     }
 
+    static String bin(AndroidLibInstrumentationTarget target) {
```

Comment:
rename method to `libInstrumentation`

Reply:
If this is renamed, on the actual rule it will be named "bin_${target.name}" due to [this](https://github.com/uber/okbuck/blob/master/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBinaryRuleComposer.groovy#L63)